x-pack/metricbeat/module/stan/_meta/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG STAN_VERSION=0.15.1
+FROM nats-streaming:$STAN_VERSION
+
+# build stage
+FROM golang:1.13-alpine AS build-env
+RUN apk --no-cache add build-base git bzr mercurial gcc
+RUN cd src && go get github.com/nats-io/stan.go/
+RUN cd src/github.com/nats-io/stan.go/examples/stan-bench && go build .
```

Comment:
should this be checked out at $STAN_VERSION?
maybe not, just saying

Reply:
It's not the same GH repo. This is not the server repo but the one for the client.

https://github.com/nats-io/stan.go/tags
https://github.com/nats-io/nats-streaming-server/tags

What I can do is to use the current tag of this repo so as to be sure that it will not break in a future master.